src/updates/index.md

```diff
@@ -1,42 +1,174 @@
 ---
 layout: layouts/default.njk
-title: Meeting Notes
+title: Updates
 ---
+
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6 float-left mr-2 mt-8">
+  <path stroke-linecap="round" stroke-linejoin="round" d="M2.25 18.75a60.07 60.07 0 0115.797 2.101c.727.198 1.453-.342 1.453-1.096V18.75M3.75 4.5v.75A.75.75 0 013 6h-.75m0 0v-.375c0-.621.504-1.125 1.125-1.125H20.25M2.25 6v9m18-10.5v.75c0 .414.336.75.75.75h.75m-1.5-1.5h.375c.621 0 1.125.504 1.125 1.125v9.75c0 .621-.504 1.125-1.125 1.125h-.375m1.5-1.5H21a.75.75 0 00-.75.75v.75m0 0H3.75m0 0h-.375a1.125 1.125 0 01-1.125-1.125V15m1.5 1.5v-.75A.75.75 0 003 15h-.75M15 10.5a3 3 0 11-6 0 3 3 0 016 0zm3 0h.008v.008H18V10.5zm-12 0h.008v.008H6V10.5z" />
+</svg> 
+
+## Financial Overview
+
+Zinc Collective, LLC publishes our Profit and Loss and Balance Sheet reports on a quarterly and annual basis as part of our commitment to public and community governance and oversight.
+
+See the latest updates <a href="https://opencollective.com/zinc-community/updates" target="_blank">here</a>.
```

Comment:
I would probably dtich here and instead go with 

>We track our income and expenses across projects publicly in our:
> - [Community Open Collective](#)
> - [Convene Open Collective](#)
> - [Moment Park Open Collective](#)
>
> Our unaudited quarterly and annual financial reports are available in our [Public Financial Reports folder](https://drive.google.com/drive/folders/1AuGzqGFKMpPxOvC2mSp1eezS_y_54wrC)

And then I would ditch all the other financial bits; so that we don't have to publish them in multiple places and try to keep them up to date.

Reply:
1. So you want me to remove the old financial data/"Finance Archives" that is in the `<details>` tag? And the PDFs?

Reply:
2. what URLs do you want these links to go to? The links above go to this PR which I assume is not the correct URLs.

> Community Open Collective
Convene Open Collective
Moment Park Open Collective

Reply:
- Moment Park https://opencollective.com/moment-park
- Convene https://opencollective.com/convene
- Community https://opencollective.com/zinc-community

Reply:
links updated in https://github.com/zinc-collective/www.zinc.coop/pull/107/commits/c338a7e2a75bf9983fd291242139676a9c6683ef and finance info and PDFs deleted in https://github.com/zinc-collective/www.zinc.coop/pull/107/commits/8ce33d1e18929426a01738ff2abcbe101911efc1